### PR TITLE
Add spawn protection in helm chart value

### PR DESCRIPTION
**Pull request by inductor**
Hello! This PR adds spawn-protection value to be editable on Helm chart.

The original feature is based on https://github.com/itzg/docker-minecraft-server/#set-spawn-protection

### CODE REVIEW

**Review by itzg — APPROVED** [review]
Thanks

File: charts/minecraft/values.yaml [hunk 1]
```diff
@@ -127,6 +127,8 @@ minecraftServer:
   spawnMonsters: true
   # Determines if villagers will be spawned.
   spawnNPCs: true
+  # Sets the area that non-ops can not edit (0 to disable)
+  spawnProtection:  16
```
Comment (itzg) [inline charts/minecraft/values.yaml:131]: I think the lint check is complaining about the extra space before 16

**Review by itzg — CHANGES_REQUESTED** [review]
Lint fix needed

File: charts/minecraft/values.yaml (hunk 1 above)
Comment (inductor) [reply charts/minecraft/values.yaml]: fixed. thanks!

**Comment by itzg** [comment]
Sorry, there was another PR ahead of yours and you'll need a small merge and resolve...and bump the version one more time.

**Comment by inductor** [comment]
I think it's a better idea to change the release number you update the version tag not on master...

**Comment by itzg** [comment]
> I think it's a better idea to change the release number you update the version tag not on master...

The current strategy is annoying in rare cases like what you experienced, but seems to be standard practice. 

@billimek any thoughts to add?

**Comment by inductor** [comment]
@itzg Hi. I updated the version. Can you merge it?

**Review by itzg — APPROVED** [review]

**Pull request merged**